server/src/main/java/org/candlepin/resource/JobResource.java

```diff
@@ -174,6 +181,42 @@ public SchedulerStatus setSchedulerStatus(boolean running) {
         return getSchedulerStatus();
     }
 
+    @POST
+    @Path("cron/{task}")
```

Comment:
@Ceiu how about "schedule_cron"? I see merit in clarifying this is for cron jobs only.